### ISSUE TITLE
SwitchListTile - Null check operator used on a null value

### ISSUE THREAD

**Issue by vasilich6107** [joanpablo/reactive_forms#196]
`ReactiveSwitchListTile` throws an error due to this line of code 
https://github.com/joanpablo/reactive_forms/blob/master/lib/src/widgets/reactive_switch_list_tile.dart#L55

```
The following _CastError was thrown building ReactiveSwitchListTile(dirty, state: ReactiveFormFieldState<bool, bool>#7ef5b):
Null check operator used on a null value

The relevant error-causing widget was: 
  ReactiveSwitchListTile file:///Users/vasilich/IdeaProjects/reactive_forms_generator/packages/reactive_forms_generator/example/lib/routes/login_nullable_form.dart:103:15
When the exception was thrown, this was the stack: 
#0      new ReactiveSwitchListTile.<anonymous closure> (package:reactive_forms/src/widgets/reactive_switch_list_tile.dart:55:33)
#1      ReactiveFormFieldState.build (package:reactive_forms/src/widgets/reactive_form_field.dart:260:27)
#2      StatefulElement.build (package:flutter/src/widgets/framework.dart:4691:27)
#3      ComponentElement.performRebuild (package:flutter/src/widgets/framework.dart:4574:15)
#4      StatefulElement.performRebuild (package:flutter/src/widgets/framework.dart:4746:11)
```

**Comment by vasilich6107** [comment]
Fix #197

**Comment by vasilich6107** [comment]
Fixed
